CHANGELOG.md

```diff
@@ -2607,4 +2607,4 @@ Features:
 
 # 1.0.0
 
-Initial release
```

Comment:
There is still a superfluous change here.

Reply:
Happy to fix if if you can tell me what on earth has changed there...

Reply:
newline at end of file I guess
should be easy to get rid of via partial (un)staging

Reply:
Is this really important to spend time on? I'd rather merge this and then someone can reformat properly on master instead. This change has 0 effect because people can't autoformat anyway.

Reply:
Yes, ok, let's go ahead